osbs/api.py

```diff
@@ -170,24 +171,34 @@ def get_pod_for_build(self, build_id):
 
     @osbsapi
     def get_build_request(self, build_type=None, inner_template=None,
-                          outer_template=None, customize_conf=None):
+                          outer_template=None, customize_conf=None,
+                          arrangement_version=5):
         """
-        return instance of BuildRequest
+        return instance of BuildRequest or BuildRequestV2
```

Comment:
Technically a `BuildRequestV2` *is* a `BuildRequest` :-) but yes, this probably makes it clearer.